README.md

```diff
@@ -36,6 +21,19 @@ page.
     </dependency>
     ```
     
+  Note: Packages can be manually installed in a local maven repository by 
```

Comment:
I'm not convinced that this information is helpful. I don't see other, maven-based projects adding this information and it could lead people to think it's required even if it's clearly optional.

Reply:
This is to bridge the gap until packages are available in Maven central.

Reply:
There should be no gap in this case. You don't need to officially publish to GitHub until we've got the artifacts in artifactory and then maven.